Assets/Scripts/Core/Interactables/Vent.cs

```diff
@@ -32,13 +32,11 @@ public class Vent : MonoBehaviour, IInteractable {
 				unit = player
 			};
 			combatInstance.OnUnitActionPointsChanged?.Invoke(this, unitEvent);
-			combatInstance.UpdateValidMovePositions();
```

Comment:
I assume you tested this?

Reply:
Yes, as it ends the turn it does not need to be here because the new turn will update it.

Reply:
Ah I understand, this looks good to me then.